docs/development.md

```diff
@@ -42,6 +52,82 @@ Use Ctrl-C to stop
 
 Default username and password for admin are: `admin@example.com` and `test123`.
 
+### Running the extension as an engine in a Rails app
```

Comment:
Same as before. Shouldn't it be `#` x 4? :slightly_smile:

Reply:
I lowered the headings.